Editor/EditorCore/ProBuilderEditor.cs

```diff
@@ -499,16 +502,21 @@ void OnGUI()
             }
         }
 
-        void Menu_ToggleIconMode()
+        void IconModeChanged()
         {
-            s_IsIconGui.value = !s_IsIconGui.value;
             if (s_EditorToolbar != null)
                 DestroyImmediate(s_EditorToolbar);
             s_EditorToolbar = ScriptableObject.CreateInstance<EditorToolbar>();
```

Comment:
I would move these three lines into one "OpenEditor()" definition as we have the same scope in InitGUI().

Reply:
Changing